### PR TITLE
Python for everybody line numbers

### PR DESCRIPTION
**Pull request by borntofrappe**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Closes #XXXXX

Following @scissorsneedfoodtoo advice, the pull request add lines numbers to the code block of those challenges which directly ask for a line number

### CODE REVIEW

File: curriculum/challenges/english/07-scientific-computing-with-python/python-for-everybody/iterations-loop-idioms.english.md [hunk 1]
```diff
@@ -20,14 +20,14 @@ question:
...
-            smallest = itervar
-            break
-        print("Loop:", itervar, smallest)
-    print("Smallest:", smallest)
+    1 smallest = None
+    2 print("Before:", smallest)
+    3 for itervar in [3, 41, 12, 9, 74, 15]:
+    4     if smallest is None or itervar < smallest:
+    5         smallest = itervar
+    6         break
```
Comment (Sky020) [inline curriculum/challenges/english/07-scientific-computing-with-python/python-for-everybody/iterations-loop-idioms.english.md:28]:
```suggestion
    5        smallest = itervar
    6        break
```

Comment (Sky020) [inline curriculum/challenges/english/07-scientific-computing-with-python/python-for-everybody/iterations-loop-idioms.english.md:28]:
```suggestion
    5        smallest = itervar
    6        break
```

**Review by ShaunSHamilton — CHANGES_REQUESTED** [review]
Small alignment fix

**Comment by borntofrappe** [comment]
I might have been too quick to update the changes. I understand the alignment issue, but the number of indents is what matters most in python.

```python
4     if smallest is None or itervar < smallest:
5         smallest = itervar
```

One space to separate the number column, and then 4 or 8 depending on the level of indentation.

What do you think @Sky020?

**Comment by Sky020** [comment]
@borntofrappe , You are right. I did not ensure there were **4** spaces (two tabs) between blocks. Thank you, for spotting that.

Hold off on fixing anything just now; let us see what is coming of #38981 . Just to prevent any unnecessary merge conflicts.

**Comment by ahmadabdolsaheb** [comment]
Thanks for bringing this issue to our attention. I believe line numbers have been implemented globally for code blocks, so we could close this PR for now? 

image

**Comment by ojeytonwilliams** [comment]
Thanks for the PR @borntofrappe, but, as Ahmad pointed out, line numbers are now being provided globally by the Prismjs plugin.

**Pull request closed without merging**